### DISCUSSION TITLE
trace-samepler-cpu-policy-plugin   overrides conflict

### DISCUSSION THREAD

**Discussion by JoyseeKing** [apache/skywalking#11396] [General]
agent version:8.16.0
plugin name:trace-samepler-cpu-policy-plugin 
question: My Springboot project threw the following error during startup and was unable to connect to the backend. Is there any other solution besides removing this plugin
console log:
org.apache.skywalking.apm.agent.core.boot.ServiceConflictException: Service class org.apache.skywalking.apm.plugin.cpu.policy.TraceSamplerCpuPolicyExtendService overrides conflict, exist more than one service want to override :class org.apache.skywalking.apm.agent.core.sampling.SamplingServicorg.apache.skywalking.apm.agent.core.boot.ServiceConflictException: Service class org.apache.skywalking.apm.plugin.cpu.policy.TraceSamplerCpuPolicyExtendService overrides conflict, exist more than one service want to override :class org.apache.skywalking.apm.agent.core.sampling.SamplingServic

**Comment by wu-sheng** [comment]
`TraceSamplerCpuPolicyExtendService` and `SamplingService`, you have to pick one. They can't work at the same time.

**Comment by JoyseeKing** [comment]
> `TraceSamplerCpuPolicyExtendService` and `SamplingService`, you have to pick one. They can't work at the same time.
I understand, thank you
